core-rust/mesh-api-server/src/mesh_api/handlers/account_balance.rs

```diff
@@ -114,44 +127,47 @@ fn get_all_balances(
 
 fn get_requested_balances(
     mapping_context: &MappingContext,
-    database: &StateManagerDatabase<impl ReadableRocks>,
+    database: &dyn SubstateDatabase,
```

Comment:
Minor: `&impl SubstateDatabase` is slightly more efficient

Reply:
Done